lib/createsend/journey_email.py

```diff
@@ -0,0 +1,87 @@
+from __future__ import absolute_import
+
+import json
+
+from createsend.createsend import CreateSendBase
+from createsend.utils import json_to_py
+
+
+class JourneyEmail(CreateSendBase):
+    """Represents a journey and associated functionality"""
+
+    def __init__(self, auth=None, journey_email_id=None):
+        self.journey_email_id = journey_email_id
+        super(JourneyEmail, self).__init__(auth)
+
+    def bounces(self, date=None, page=None, page_size=None, order_direction=None):
+        """Retrieves the bounces for this journey email."""
+        params = {}
+        if date is not None:

```

Comment:
These block's of _if_ statements are duplicates. My suggestion is to extract a method and consume it in each of these.